packages/cli/bin/designsystemet.ts

```diff
@@ -38,14 +38,17 @@ function makeTokenCommands() {
     .option('-b1, --brand1 <number>', `Brand1 hex color`)
     .option('-b2, --brand2 <number>', `Brand2 hex color`)
     .option('-b3, --brand3 <number>', `Brand3 hex color`)
```

Comment:
Maybe define the color options as `requiredOption` instead of `option` while you're at it?

Reply:
Good point